Makefile

```diff
@@ -1,15 +1,22 @@
-DIRECTORY := integration
-DRIVER    := hetznercloud
+SHELL := bash
 
-default: test
+venv:
+	python3 -m venv venv
+	venv/bin/pip install -e .[test]
 
-clean:
-	@rm -rf $(DIRECTORY)
+export ANSIBLE_COLLECTIONS_PATH = $(shell pwd)/ansible_collections
+
+ansible_collections: venv
+	venv/bin/ansible-galaxy collection install -r requirements.yml
```

Comment:
I think this needs a `PHONY` marker, otherwise the collections are not updated if the requirements change or if the directory already exists.

`ansible-galaxy collection install` already checks if the installed collections are up-to-date and only does something if something needs to be done.

Reply:
I agree that we might be out of sync at some point if we don't clean the project once in a while.

The integrations test only relies on the collections being installed, not being up-to-date. And this might add some overhead when running the integrations test over and over.

I am never sure what is the best way to do this, as we use make in a weird way here.

Reply:
I had a local issue where the `ansible_collections` directory was created but no collections were installed. This lead to test failures for me. 

Deferring to your expertise here :)